README.md

```diff
@@ -22,4 +22,4 @@ Device specific JSON templates for RevPi HAT EEPROM generation
 | [RevPi Connect 4 WLAN 32/8GB](./revpi-hat-PR100380R00.json) | KUNBUS GmbH | PR100380 | R00 | 0 | revpi-connect4 |
 
 
-**Pleae note that this README file is auto-generated by the `scripts/update-readme.py` script.**
\ No newline at end of file
+**Pleae note that this README file is auto-generated by the `scripts/update-readme.py` script. More details can be found [here](scripts/README.md)**
```

Comment:
Pleae -> Please
`**Please note that this README file is auto-generated by the `scripts/update-readme.py` script. More details can be found [here](scripts/README.md)**`